src/js/mixin/resizeTable.js

```diff
@@ -0,0 +1,32 @@
+export default {
+    methods: {
+        resizeTable () {
+            let thead, startOffset;
+
+            document.querySelectorAll("table th").forEach(th => {
+                th.style.position = 'relative';
+
+                var grip = document.createElement('div');
+                grip.classList.add('resizeTable');
+                grip.innerHTML = "&nbsp;";
+
+                grip.addEventListener('mousedown', function (e) {
+                    thead = th;
+                    startOffset = th.offsetWidth - e.pageX;
+                });
+
+                th.appendChild(grip);
+
+                document.addEventListener('mousemove', function (e) {
```

Comment:
Place this `addEventListener` inside of `grip@mousedown` and the `grip@mouseup` clears the `thead`